Refactor the DataGrid and PivotGrid export snippets (#1235)
DevExpress/devextreme-documentation: api-reference/10 UI Widgets/dxPivotGrid/1 Configuration/export/export.md

            return (
                <PivotGrid ... >
                    <Export enabled={true} />
                </PivotGrid>
            );
        }

    ---

3. **Export the PivotGrid**   

>     Шmplement the [onExporting](/Documentation/ApiReference/UI_Widgets/dxPivotGrid/Configuration/#onExporting) handler and call the [exportPivotGrid(options)](/Documentation/ApiReference/Common/Utils/excelExporter/#exportPivotGridoptions) method in it. In the code below, this method exports the **PivotGrid** as is, but you can use [ExportPivotGridProps](/Documentation/ApiReference/Common/Object_Structures/ExportPivotGridProps/) to configure export settings, including [cell customization](/Documentation/ApiReference/Common/Object_Structures/ExportPivotGridProps/#customizeCell). To save the Excel document, call the FileSaver's **saveAs** method. The `e.cancel` parameter disables the deprecated built-in export implementation with fewer capabilities.

Reviewer: ```suggestion
    Implement the [onExporting](/Documentation/ApiReference/UI_Widgets/dxPivotGrid/Configuration/#onExporting) handler and call the [exportPivotGrid(options)](/Documentation/ApiReference/Common/Utils/excelExporter/#exportPivotGridoptions) method in it. In the code below, this method exports the **PivotGrid** as is, but you can use [ExportPivotGridProps](/Documentation/ApiReference/Common/Object_Structures/ExportPivotGridProps/) to configure export settings, including [cell customization](/Documentation/ApiReference/Common/Object_Structures/ExportPivotGridProps/#customizeCell). The **PivotGrid** is exported to an Excel worksheet that is created using the ExcelJS API. To save the Excel document, call the FileSaver's **saveAs** method. The `e.cancel` parameter disables the deprecated built-in export implementation with fewer capabilities.
```

                        excelCell.alignment = { horizontal: 'left' };
                    } 
                }).then(function() {

>                     workbook.xlsx.writeBuffer().then(function(buffer: BlobPart) { 

Reviewer: ```suggestion
                    workbook.xlsx.writeBuffer().then(function(buffer) { 
```

                    } 
                }).then(function() {
                    workbook.xlsx.writeBuffer()

>                         .then(function(buffer: BlobPart) {

Reviewer: The `BlobPart` type should be specified in all Angular examples with this code.

                        } 
                    }).then(function() {
                        workbook.xlsx.writeBuffer()

>                             .then(function(buffer: BlobPart) {

Reviewer: ```suggestion
                            .then(function(buffer) {
```

                } 
            }).then(function() {
                workbook.xlsx.writeBuffer()

>                     .then(function(buffer: BlobPart) {

Reviewer: ```suggestion
                    .then(function(buffer) {
```
